staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -77,8 +77,6 @@ type Event struct {
 	// AuditLevel at which event was generated
 	Level Level
 
-	// Time the request reached the apiserver.
-	Timestamp metav1.Time
```

Comment:
I don't think it matters which timestamp we use for which, as long as it's clearly documented. I have a slight preference for Creation timestamp being the "request received" timestamp because that's when we initialize the audit event struct.